mail_tracking/static/src/js/mail_tracking.js

```diff
@@ -140,24 +141,28 @@ odoo.define('mail_tracking.partner_tracking', function (require) {
             this.do_action(action);
         },
         on_tracking_status_click: function (event) {
-            var tracking_email_id = $(event.currentTarget).data('tracking');
-            var state = {
-                'model': 'mail.tracking.email',
-                'id': tracking_email_id,
-                'title': _t("Message tracking"),
-            };
-            event.preventDefault();
-            this.action_manager.do_push_state(state);
-            var action = {
-                type:'ir.actions.act_window',
-                view_type: 'form',
-                view_mode: 'form',
-                res_model: 'mail.tracking.email',
-                views: [[false, 'form']],
-                target: 'new',
-                res_id: tracking_email_id,
-            };
-            this.do_action(action);
+            // Only admins can check advanced tracking stuff
+            session.user_has_group('base.group_system').then(has_group => {
```

Comment:
Why this limitation? It's very useful for people to check the tracking lines.

Reply:
I took a new approach dinamically evaluating acls the same way mail.message does relaying on the related records permissions to filter the trackings a user can see.